src/content/blog/2025-01-22-eslint-2024-year-review.md

```diff
@@ -0,0 +1,156 @@
+
+Towards the last quarter of 2024, things had started to settle and we went back to work on improving the configuration system to address some of the common issues users reported.
+
+## Release of the language plugins API
+
+Part of the move to the new configuration system was with an eye towards making ESLint language-agnostic through the creation of [language plugins](https://eslint.org/blog/2024/07/whats-coming-next-for-eslint/#language-plugins). This involved a lot of core work to move the JavaScript-specific functionality into its own classes, which in turn allowed us to swap in similar class for other languages.
+
+In the end, we released official language plugins for [Markdown](https://github.com/eslint/markdown) and [JSON](https://github.com/eslint/json), as well as a prototype for [CSS](https://github.com/eslint/css). All language plugins allow you to create and distribute your own rules that run against the language-specific ASTs. To facilitate the creation of these custom rules, we also introduced the [Code Explorer](https://explorer.eslint.org).
+
+## Updates to support the latest ECMAScript features
+
+As with most years, the team spends a lot of time monitoring the JavaScript standardization process to ensure ESLint is updated to support new syntax as quickly as possible. In 2024 we added support for the following new JavaScript features:
+
+* [ES2025 Regex duplicate named capture groups](https://github.com/tc39/proposal-duplicate-named-capturing-groups)
+* [ES2025 Regex modifiers](https://github.com/tc39/proposal-regexp-modifiers)
+* [ES2025 Import attributes](https://github.com/tc39/proposal-import-attributes)
+
+These features involved changes not just to parsing but also to rules that work with regular expressions and modules.
+
+## Financials review
+
+The next few sections dig into the project financials. All of this information is gathered from the Open Collective transaction list with the exception of Tidelift payments (which are retrieved from Tidelift directly). Because transactions come in all throughout the course of a day, we decided to consider transactions beginning on January 1, 2024 at midnight UTC and ending on December 31, 2024 at 11:59 UTC as our fiscal 2024.
```

Comment:
```suggestion
The next few sections dig into the project financials. All of this information is gathered from the Open Collective transaction list with the exception of Tidelift payments (which are retrieved from Tidelift directly). Because transactions come in all throughout the course of a day, we decided to consider transactions beginning on January 1, 2024 at midnight UTC and ending on December 31, 2024 at 11:59 p.m. UTC as our fiscal 2024.
```